.github/workflows/spark-integration-tests.yml

```
@@ -0,0 +1,294 @@
+        cd other/java/hdfs2
+        mvn clean install -DskipTests -Dgpg.skip=true
+        cd ../../..
+        
+        echo "Building HDFS3 client..."
+        cd other/java/hdfs3
+        mvn clean install -DskipTests -Dgpg.skip=true
+        echo "✓ All Java dependencies built"
+
+    - name: Cache Apache Spark
+      id: cache-spark
+      uses: actions/cache@v3
```

actionlint (1.7.8)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)